### DISCUSSION TITLE
Reset catalog classification

### DISCUSSION THREAD

**Discussion by jp136769** [r-lidar/lidR#407] [Q&A] [answered]
Hello,

I don't know if I'm in the right section, fell free to indicate me another channel in case of need.
Is there a way the reset the entire classification of each point from `LAScatalog` (from whatever to 0) ? `classify_ground()` does it for class-2 points, I imagine it must be possible for the full dataset!

Thanks,

**Comment by Jean-Romain** [comment, marked as answer]
How the hell did you open a "discussion" thread? I didn't even know this feature existed. Is it a new github feature?

I no longer answer to questions about lidR by email or on github because I receive too many questions from too many users. If you have questions about using the lidR package please ask on [gis.stackexchange.com](https://gis.stackexchange.com/) with the tag `lidr`. On stackexchange.com other users can also answer your questions and the answers thus provided will be public and well indexed by search engines so they can also help other users. Moreover we can provide link to already answered questions.

In your case the question being very simple I can make an exception :wink: 

```r
las$Classification <- 0L
```

You can also reset the classification on the fly at read time

```r
las = readLAS(LASfile, filter = "-set_classification 0")
```

**Reply by jp136769** [reply]
Never used this feature before, I simply clicked on New discussion, within Discussion tab, between Pull requests and Actions on your lidR main page! All right I'll post on gis.stack* next time.
 
Clear answer, thank for the exception ;)

**Reply by Jean-Romain** [reply]
I'm pretty sure this new tab did not exist yesterday. With this new feature, asking question on github seems a fair option. But I still prefer stackexchange because I'm sure it is well referenced by search engines and because other people can answer as well.
